packages/auto-install/rollup.config.js

```diff
@@ -4,10 +4,10 @@ import pkg from './package.json';
 
 export default {
   input: 'src/index.ts',
-  plugins: [typescript()],
+  plugins: [typescript({ sourceMap: false })],
```

Comment:
why are we removing the sourcemaps?

Reply:
They were never present. Rollup needs to have the sourcemap flag for them to be generated. But the new TypeScript plugins warns when the Rollup and plugin options do not match. Honestly I think it is ok for libraries to not ship sourcemaps, but of course we could consider adding them for our own testing if we want.